packaging/nntrainer.spec

```diff
@@ -103,6 +104,12 @@ Requires:	nnstreamer-tensorflow-lite
 Requires:	%{capi_machine_learning_inference}
 %endif # support_nnstreamer_backbone
 
+%if 0%{?support_tflite_backbone}
+BuildRequires: tensorflow-lite-devel
```

Comment:
In Tizen, I'd recommend tensorflow2-lite-devel

Reply:
currently, nntrainer depends on tensorflow-lite and nnstreamer-tensorflow-lite, and tf-lite for android as well.
I will update both to tf2-lite for all together in a separate PR - raised #1244.

Reply:
added #1245